features/bootstrap/FeatureContext.php

```diff
@@ -67,6 +67,22 @@ public function alterUserParameters(EntityScope $scope) {
     }
   }
 
+    /**
+     * Hook into user creation to test `@beforeEntityCreate`
+     *
+     * @beforeEntityCreate
+     */
+    public function alterEntityParameters(EntityScope $scope)
+    {
+        // @see `features/entity.feature`
+        // Concatenate 'First name' and 'Last name' to form user name.
+        $entity = $scope->getEntity();
+        if (isset($entity->{"First name"}) && isset($entity->{"Last name"})) {
+            $entity->name = $entity->{"First name"} . ' ' . $entity->{"Last name"};
+            unset($entity->{"First name"}, $entity->{"Last name"});
+        }
```

Comment:
This has nothing to do with nodes, nodes don't have a name field. This is a user. also, the method would be getEntityTypeId(), not bundle. bundle is article, page, ...